docs/cloud-provider-config.md

```diff
@@ -42,6 +42,8 @@ Note: All values are of type `string` if not explicitly called out.
 |userAssignedIdentityID|The Client ID of the user assigned MSI which is assigned to the underlying VMs|Required for user-assigned managed identity.|
 |subscriptionId|The ID of the Azure Subscription that the cluster is deployed in|**Required**.|
 |identitySystem|The identity system for AzureStack. Supported values are: ADFS|Only used for AzureStack|
+|networkResourceTenantID|The AAD Tenant ID for the Subscription that the network resources are deployed in|Optional. Only used for hosting network resources in different AAD Tenant and Subscription than those for the cluster.|
```

Comment:
Could you add version info for it? e.g. `Optional. Supported since v1.18.0...`

Reply:
So sorry @feiskyer I was busy with some stuff and missed the comment. Saw you helped to refine the statement already. Thank you!